daemon/inspect.go

```diff
@@ -238,3 +240,15 @@ func (daemon *Daemon) getDefaultNetworkSettings(networks map[string]*network.End
 	}
 	return settings
 }
+
+func getContainersWithVolume(volume string, containers []*container.Container) []string {
+	var containerNames []string
+	for _, container := range containers {
```

Comment:
This double iteration is going to make the command very slow when you have lots and lots of containers.